development.postgresql.yml

```diff
@@ -40,6 +40,7 @@ services:
       - mdb 
     volumes:
       - ./notarysql/postgresql-initdb.d:/docker-entrypoint-initdb.d
+    command: -l
```

Comment:
Per https://www.postgresql.org/docs/9.5/static/app-postgres.html, `-l` "Enables secure connections using SSL.". 

FWIW, I think `ssl=on` is implicit after passing `-l`, but since I was editing the conf for `ssl_ca_file` path, I thought it would be clearer to set `ssl=on` explicitly, in case anything changes silently in the future versions. (e.g. Not sure what happens if we use `-l` and `ssl=off` -- which one takes a higher priority is never mentioned. :( In fact, the docs that mention `ssl-on` never refer to `-l`, and vice versa.)